### PR TITLE
fix: normalize line height to sb-scale-ratio

### PR DESCRIPTION
**Pull request by thefrontendwizard**
JIRA ticket: https://usabilla.atlassian.net/browse/BLUP-244

Preview:
https://static-staging.usabilla.com/stylabilla/pr/fix-line-height/docs/section-style.html#kssref-style-typography-headings
https://static-staging.usabilla.com/stylabilla/pr/fix-line-height/docs/section-components.html#kssref-components-card

## What to test? Scope of the test?

Normalize line height across application to scale-ratio (1.2)

## Reviews

Reviewers: @chiel @MLKiiwy @thefrontendwizard @kagnamazian @realdreamer @moroale93 @DavideBruner @ranst91-u @galagmon @Maruceau @gsbkr @michelvdhoek

### CODE REVIEW

**Comment by galagmon** [comment]
@thefrontendwizard not sure what I am testing

File: src/components/_tooltip.scss [hunk 1]
```diff
@@ -121,7 +121,7 @@ $arrow-height: sb-px2rems(4px);
 
     .fill {
       /* stylelint-disable-next-line declaration-no-important */
-      fill: $sb-color-white !important;
+      fill: $sb-color-white;
```
Comment (realdreamer) [inline src/components/_tooltip.scss]: this change is it really needed here? because your commit changes says about only `font-size` to `scale`

Comment (thefrontendwizard) [reply src/components/_tooltip.scss]: indeed, it sneaked it's way in 😄 
it's out of context. sorry about that.

**Comment by chiel** [comment]
Please squash before merge btw. :)

**Pull request merged**